GoogleDataTransport/GDTCORTests/Unit/GDTCORStorageTest.m

```diff
@@ -275,35 +279,32 @@ - (void)testStoreMultipleEvents {
   });
 }
 
-/** Tests enforcing that a prioritizer does not retain an event in memory. */
+/** Tests enforcing that a prioritizer does not retain the DataObjectTransportBytes of an event in
+ * memory.
+ */
 - (void)testEventDeallocationIsEnforced {
-  __weak GDTCOREvent *weakEvent;
-  GDTCORStoredEvent *storedEvent;
-  @autoreleasepool {
-    GDTCOREvent *event = [[GDTCOREvent alloc] initWithMappingID:@"404" target:target];
-    weakEvent = event;
-    event.dataObjectTransportBytes = [@"testString" dataUsingEncoding:NSUTF8StringEncoding];
-    event.clockSnapshot = [GDTCORClock snapshot];
-    // Store the event and wait for the expectation.
-    XCTestExpectation *writtenExpectation = [self expectationWithDescription:@"event written"];
-    XCTAssertNoThrow([[GDTCORStorage sharedInstance] storeEvent:event
-                                                     onComplete:^(BOOL wasWritten, NSError *error) {
-                                                       XCTAssertTrue(wasWritten);
-                                                       XCTAssertNil(error);
-                                                       [writtenExpectation fulfill];
-                                                     }]);
-    [self waitForExpectations:@[ writtenExpectation ] timeout:10.0];
-    GDTCORDataFuture *dataFuture =
-        [[GDTCORDataFuture alloc] initWithFileURL:[NSURL fileURLWithPath:@"/test"]];
-    storedEvent = [event storedEventWithDataFuture:dataFuture];
-  }
+  __weak NSData *weakDataObjectTransportBytes;
+  GDTCOREvent *event = [[GDTCOREvent alloc] initWithMappingID:@"404" target:target];
```

Comment:
Similar to how the test was written before, wrap in an autoreleasepool because otherwise there will be a race condition in this test that will manifest as flakiness. (Sometimes the event will be deallocated, sometimes it won't--it'll be a race between dealloc and the dealloc check.)